src/main/java/com/internship/backend/service/ReservationService.java

```diff
@@ -53,11 +68,11 @@ public Reservation addReservation(Reservation reservation) throws ReservationAlr
     public boolean isValidReservation(Reservation reservation){
```

Comment:
Can you add some unit tests for this method? In general methods defined as public should be tested.  And also, could this method be private?